### PR TITLE
addition of a .travis.xml file and a twick to the index.js

### PR DESCRIPTION
**Pull request by maestro-1**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+language : node_js
+
+node_js: node_js
+
+cache: npm
+
+install:
+  -npm ci
```
Comment (thatkazuk1) [inline .travis.yml:10]: This should be `npm i` or `npm install`

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+language : node_js
+
+node_js: node_js
+
+cache: npm
```
Comment (thatkazuk1) [inline .travis.yml]: `cache`: `node_modules`

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  -npm ci
+
+script:
+  -npm test
+
+deploy:
+  provider: pages
+  github-token: $GITHUB_TOKEN
+  on:
+    branch: master
```
Comment (thatkazuk1) [inline .travis.yml]: Add a newline here or better still add this line to your VSCode `settings.json` file `  "files.insertFinalNewline": true,`

**Review by thatkazuk1 — CHANGES_REQUESTED** [review]
Please effect those changes @maestro-1

File: src/index.js [hunk 4]
```diff
@@ -1,36 +1,44 @@
...
+})
+
+app.get((error, req, res) => {
+  res.status(error.status || 500);
+  res.json({
+    error:{
+      message : error.message
+    }
+  })
+});
```
Comment (thatkazuk1) [inline src/index.js]: Hmmm🤔

File: .travis.yml (hunk 2 above)
Comment (maestro-1) [reply .travis.yml]: error on my part, accidentally filled out npm instead of node_modules

File: src/index.js (hunk 4 above)
Comment (maestro-1) [reply src/index.js]: error.message should be an edited string but this way of handling errors ensures that it accounts for both 404 and 500 errors and anything inbetween

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  - npm run coverage
+
+notifications:
+  email: false    
+
+deploy:
+  provider: pages
+  github-token: $GITHUB_TOKEN
+  on:
+    branch: master
```
Comment (thatkazuk1) [inline .travis.yml]: We are working with `develop` branch

File: src/index.js (hunk 4 above)
Comment (thatkazuk1) [reply src/index.js]: 👍

File: .travis.yml (hunk 5 above)
Comment (maestro-1) [reply .travis.yml]: okay

**Review by thatkazuk1 — APPROVED** [review]
👍

**Pull request merged**